scipy/signal/_filter_design.py

```diff
@@ -2942,7 +2942,9 @@ def butter(N, Wn, btype='low', analog=False, output='ba', fs=None):
     Parameters
     ----------
     N : int
-        The order of the filter.
+        The order of the filter. For bandpass filters, the resulting order
+        of the final sos matrix is 2*N, so N represents the number of
+        biquad sections of the desired system.
```

Comment:
And for the other types of filter it's the order?

Also this
```suggestion
        The order of the filter. For bandpass filters, the resulting order
        of the final second-order sections ('sos') matrix is ``2*N``,
        with `N` the number of biquad sections of the desired system.
```

Reply:
For `'lowpass'` and `'highpass'` `N` is actually `N`. Good improvement. Sorry for the unnecessary CI, never contributed to `scipy` before.

Reply:
And bandstop?

No problem for the CI, I know it's your first PR 😃